### PR TITLE
Field style fixes

### PR DESCRIPTION
**Pull request by dbanksdesign**
*Issue #, if available:*

*Description of changes:*

* Removing 'hacks' in docs CSS to fix some visual issues with fields
* Added explicit tokens for field label, field message, and input color

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/src/pages/components/alert/AlertPropControls.tsx [hunk 1]
```diff
@@ -36,9 +35,8 @@ export const AlertPropControls: AlertPropControlsInterface = ({
   body,
   setBody,
 }) => {
-  const { tokens } = useTheme();
   return (
-    <>
+    <Flex direction="column">
```
Comment (dbanksdesign) [inline docs/src/pages/components/alert/AlertPropControls.tsx:39]: Removed `.amplify-field { margin-bottom }` style, so just remember to wrap forms in a Flex.

File: docs/src/styles/docs/main.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.docs-main p {
+.docs-main p:not([class*='amplify']) {
```
Comment (dbanksdesign) [inline docs/src/styles/docs/main.scss:1]: We want paragraph tags from MDX content to have a bottom margin, but this was messing up a lot of our components that render p tags.

File: docs/src/styles/docs/overrides.scss [hunk 3]
```diff
@@ -6,39 +6,8 @@
   border-color: var(--amplify-colors-border-primary);
 }
 
-.amplify-field-group :not(:last-child) .amplify-input {
-  border-end-end-radius: var(--amplify-components-fieldcontrol-border-radius);
```
Comment (dbanksdesign) [inline docs/src/styles/docs/overrides.scss:10]: TBH dunno why this was ever in there.

File: docs/src/styles/docs/overrides.scss [hunk 4]
```diff
@@ -6,39 +6,8 @@
...
-}
-
-.amplify-field-group__outer-end {
-  position: relative;
-}
-
-
-.amplify-label,
-.amplify-switch__wrapper {
-  color: var(--amplify-colors-font-secondary);
```
Comment (dbanksdesign) [inline docs/src/styles/docs/overrides.scss:21]: This is now a design token.

File: docs/src/styles/docs/overrides.scss [hunk 5]
```diff
@@ -6,39 +6,8 @@
...
-  box-sizing: content-box;
-}
 
 .amplify-button[data-variation='primary'] {
   border-color: var(--amplify-components-button-primary-border-color);
 }
 
-.amplify-select,
-.amplify-input {
-  color: var(--amplify-components-fieldcontrol-color);
```
Comment (dbanksdesign) [inline docs/src/styles/docs/overrides.scss:43]: This is now a design token.

File: packages/ui/src/theme/css/component/field.scss [hunk 6]
```diff
@@ -11,12 +11,8 @@
     font-size: var(--amplify-components-field-large-font-size);
     gap: var(--amplify-components-field-large-gap);
   }
+}
 
-  // Required to override docs styling
-  // css-s90uia p { margin-bottom: 1rem; }
-  // @TODO: remove styling from docs
-  .amplify-field__description,
```
Comment (dbanksdesign) [inline packages/ui/src/theme/css/component/field.scss:18]: Fixed this.

File: packages/ui/src/theme/tokens/components/fieldMessages.js [hunk 7]
```diff
@@ -1,9 +1,10 @@
 module.exports = {
   error: {
-    color: { value: '{colors.border.error.value}' },
+    color: { value: '{colors.font.error.value}' },
```
Comment (dbanksdesign) [inline packages/ui/src/theme/tokens/components/fieldMessages.js:3]: This is a font color so it should be a font color.

File: docs/src/pages/components/alert/AlertPropControls.tsx (hunk 1 above)
Comment (reesscot) [reply docs/src/pages/components/alert/AlertPropControls.tsx:39]: https://media.giphy.com/media/3WCNY2RhcmnwGbKbCi/giphy.gif

File: packages/ui/src/theme/css/component/field.scss (hunk 6 above)
Comment (reesscot) [reply packages/ui/src/theme/css/component/field.scss:18]: fVtcfEXWQJQUbsF1sH

**Review by reesscot — APPROVED** [review]
Ship it

**Pull request merged**